Assets/Scripts/UI/CameraIntMsg.cs

```diff
@@ -0,0 +1,47 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class CameraIntMsg : MonoBehaviour {
+
+    public GameObject Camera;
+    public GameObject ArrowsSide;
+    public GameObject ArrowsUp;
+    public Controller2 isghostMode;
```

Comment:
so that I can check if it is in ghost mode, its better than saying is x is pressed, i can change the name